packages/modelica_ml/modelica_ml.0.2.0/opam

```diff
@@ -0,0 +1,27 @@
+opam-version: "1.2"
+maintainer: "\"Christoph Höger <christoph.hoeger@tu-berlin.de>\""
+authors: "\"Christoph Höger <christoph.hoeger@tu-berlin.de>\""
+homepage: "http://github.com/choeger/modelica.ml"
+license: "BSD-3-clause"
+bug-reports: "https://github.com/choeger/modelica.ml/issues"
+dev-repo: "https://github.com/choeger/modelica.ml.git"
+build: [
+  ["ocaml" "setup.ml" "-configure" "--prefix" prefix]
+  ["ocaml" "setup.ml" "-build"]
+]
+install: ["ocaml" "setup.ml" "-install"]
+build-doc: ["ocaml" "setup.ml" "-doc"]
+remove: ["ocamlfind" "remove" "modelica_ml"]
+depends: [
+  "batteries"
+  "menhir"
+  "ocamlfind"
+  "ocamlgraph"
+  "ppx_deriving" {= "1.0"}
+  "ppx_deriving_yojson" {= "2.0"}
```

Comment:
These equality constraints are probably too tight and will cause issues. Both packages have newer version. If you want to constrain to `1.0.x` or `2.0.x`, you need to use lower and upper bounds (probably inclusive lower and exclusive upper).